src/montagu_deploy/config.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, path, extra=None, options=None):
         self.proxy_ref = self.build_ref(dat, "proxy")
         self.proxy_port_http = config.config_integer(dat, ["proxy", "port_http"])
         self.proxy_port_https = config.config_integer(dat, ["proxy", "port_https"])
+        self.proxy_port_metrics = config.config_integer(dat, ["proxy", "port_metrics"], is_optional=True, default=9000)
```

Comment:
I'm not sure how useful these being configurable even is, since presumably the container will only ever listen on 80/443/9000. The only usefulness would be if we mapped a different host port to those, eg setting `port_http` to `8080` would bind host port 8080 to container port 80, but at a glance that doesn't seem to be how it behaves. 

Anyway that kind of irrelevant to this change.

Reply:
Yeah, agreed, not sure what the intention was originally.